tests/optimizer.py

```diff
@@ -28,11 +28,13 @@ class TestOptimizers(unittest.TestCase):
     def __init__(self, methodname):
         """Initialize the test class."""
         super().__init__(methodname)
-        self.num_steps = 100
+        self.num_steps = 200
```

Comment:
Why make it 200? Is it going to double the runtime of the test?

Reply:
Switched back to 100 after making the LARS test succeed within 100 steps